README.md

```diff
@@ -238,7 +238,7 @@ solution).
 
 The build of Raspbian is divided up into several stages for logical clarity
 and modularity.  This causes some initial complexity, but it simplifies
-maintenance and allows for more easy customization.
+maintenance and allows for easier customization.
```

Comment:
I suppose the really correct wording would be: "and allows customizations to be more easily made."